packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -744,8 +744,8 @@ test.serial('writeGranulesFromMessage() on re-write saves granule records to Dyn
     granule_id: granuleId,
     collection_cumulus_id: collectionCumulusId,
   });
-  const apiFormattedPostgresGranule =
-    await translatePostgresGranuleToApiGranule({
+  const apiFormattedPostgresGranule
+    = await translatePostgresGranuleToApiGranule({
```

Comment:
Nit: Removed prior comment - I think we should consider elevating the lint warning here wholesale to an error  - although I wouldn't mind a discussion re: this default (prettier defaults to equals on the line, at least in default config), but that's not required for this PR.

Reply:
I haven't seen the various opinions on this, so I'm not sure which side I would fall on. I was just following the suggested lint fix. I can revert these changes, just let me know.

Reply:
No, that's fine, we need to have a broader conversation about opinionated formatting options, can save for arch meeting.